app/command.go

```diff
@@ -155,10 +155,17 @@ func (a *App) ListAllCommands(teamId string, T goi18n.TranslateFunc) ([]*model.C
 }
 
 func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	parts := strings.Split(args.Command, " ")
-	trigger := parts[0][1:]
+	message := ""
+	trigger := ""
+	if strings.ContainsAny(args.Command, " \n\t\v\r\f") {
+		index := strings.IndexAny("args.Command", " \n\t\v\r\f")
```

Comment:
The `"args.Command"` doesn't seem right -- are the unit tests failing as a result?